src/databricks/labs/ucx/cli.py

```diff
@@ -273,5 +273,33 @@ def save_aws_iam_profiles(w: WorkspaceClient, aws_profile: str | None = None):
     return None
 
 
+@ucx.command
+def save_uc_compatible_roles(w: WorkspaceClient, *, aws_profile: str | None = None):
+    """extracts all the iam roles with trust relationships to the UC master role.
+     Map these roles to the S3 buckets they have access to.
+    Requires a working setup of AWS CLI.
+    https://aws.amazon.com/cli/
```

Comment:
Add this requirement in the README also

Reply:
Done

Reply:
I see that our readme doesn't reference the CLI.
I'll do that in a separate PR after consulting with @nfx